docs/sdks/architecture/deployment-modes.md

```diff
@@ -1,152 +1,292 @@
+## Patterns to avoid
 
-1. **Users not exposed to the experiment are included in the analysis.** Imagine you're running a test on the checkout page and say you have 10,000 visitors to your site each day, 500 of whom reach the checkout page. If you include all 10,000 users in your analysis, you've unlikely to see even a large change in the behavior of the 500 that hit the checkout page. This can lead to understated experiment results, and calling tests neutral even if they are actually winners.
+### Initializing the SDK each time a flag is evaluated
 
-2. **Users are more likely to be included in the experiment if they are in control**. Imagine you're experimenting on a new search algorithm and users are flagged as "exposed" once they complete a search. If a new search model has a higher latency, this could lead to more control users being included in the analysis and introduce meaningful bias in experiment results. [Sample Ratio Mismatch](/experiment-analysis/diagnostics/#traffic-imbalance-diagnostic) (SRM) tests are designed to detect this type of issue, but often use a very low p value threshold (typically 0.001). That is, it's easy to imagine a case like this hypothetical search model experiment where SRM alerts are not triggered, but experiment results are still impacted by SRM-related bias.
+A common anti-pattern is to initialize the SDK each time a flag is evaluated. This is unnecessary thanks to Eppo's local evaluation architecture. Further, this causes excessive network calls to Eppo's CDN leading to not just increased latency, but also potential breaching of Eppo's CDN request limits.
 
+Instead, initialize once at the start of the application lifecycle and use the `getInstance` method when you need to evaluate a flag.
 
-Both of these issues are in theory solvable if teams are diligent about filtering on the correct exposure event. However, as experimentation programs grow we have observed that this becomes increasingly hard to enforce and monitor. As you consider different options to deploy Eppo's SDK, make sure that you consider not just immediate implementation costs, but also long term scalability and tech debt. 
+### Initializing from CDN in a serverless function
 
-For teams that need to fetch variants at session start (say, for fully server-side rendered architectures), Eppo can support this pattern. This is done by exporting a list of flags from the SDK and evaluating each of them in a for loop. That said, for the reasons above it is highly encouraged that teams instead follow one of the other deployment patterns whenever possible.
+Similarly, make sure to follow the recommendations above for serverless architectures. If you instead make a request to Eppo's CDN each time the serverless function is called, you'll introduce unnecessary latency and risk breaching Eppo's CDN limits.
```

Comment:
By "recommendations above" you mean "pre-fetched configurations", right? Perhaps link to the section directly